README.md

```diff
@@ -27,170 +27,3 @@ After you're finished please remove all the comments and instructions!
 # 📗 Table of Contents
 
 - [📗 Table of Contents](#-table-of-contents)
```

Comment:
- [ ] I think it's better to create a README file with all the sections on your feature branch. After the approval of the content of the feature branch, you can merge it into the master. This is the work flow used by developers.

Reply:
please explain this in detail or give some sample if possible